citesphere-importer/pom.xml

```diff
@@ -15,7 +15,8 @@
         <spring-data.version>Lovelace-RELEASE</spring-data.version>
         <thymeleaf.version>3.0.11.RELEASE</thymeleaf.version>
         <spring.kafka.version>2.2.6.RELEASE</spring.kafka.version>
-        <citesphere.messages.version>0.2</citesphere.messages.version>
+        <citesphere.messages.version>0.5</citesphere.messages.version>
+        <crossref-connect-version>0.3-SNAPSHOT</crossref-connect-version>
```

Comment:
the current version of crossref-connect is 0.2. Where there changes and is there a pull request? If so, it needs to be linked. If not, this needs to be the correct version.